src/pybind/mgr/mgr_util.py

```diff
@@ -554,10 +554,13 @@ def verify_cacrt_content(crt):
     try:
         x509 = crypto.load_certificate(crypto.FILETYPE_PEM, crt)
         if x509.has_expired():
-            logger.warning('Certificate has expired: {}'.format(crt))
+            org, cn = get_cert_issuer_info(crt)
+            end_date = datetime.datetime.strptime(x509.get_notAfter().decode('ascii'), '%Y%m%d%H%M%SZ')
+            msg = f'Certificate issued by "{org}/{cn}" expired on {end_date}'
```

Comment:
+1 on checking that certs are expired; in OSP we create certs "per network", and the `hostname` is often tied to the storage network (and it's not the same of the output of `cephadm orch host ls`). I'm worried we can introduce a regression on that use case, but I don't see evidence in the code, as you're using org/cn just to populate the msg of the expired certificate (which looks good to me).

Reply:
> +1 on checking that certs are expired; in OSP we create certs "per network", and the `hostname` is often tied to the storage network (and it's not the same of the output of `cephadm orch host ls`). I'm worried we can introduce a regression on that use case, but I don't see evidence in the code, as you're using org/cn just to populate the msg of the expired certificate (which looks good to me).

I'm not sure I'm following here... right now `org/cn`  is only used to populate the log message. Do you need this information for other purpose?

Reply:
No, it's fine, as long as it's used for this purpose LGTM